src/main/java/uk/gov/companieshouse/web/accounts/service/payment/impl/PaymentServiceImpl.java

```diff
@@ -49,33 +44,20 @@ public PaymentServiceImpl(ApiClientService apiClientService, EnvironmentReader e
      * {@inheritDoc}
      */
     @Override
-    public String createPaymentSessionForTransaction(String transactionId, String paymentUrl)
+    public String createPaymentSessionForTransaction(String transactionId)
             throws ServiceException {
 
+        String paymentUrl = "/payment";
```

Comment:
Can we make this a `private static final String` defined on the class? There's no need to create a new `paymentUrl` every time this method is called.